packages/atomic/src/components/search/atomic-notifications/atomic-notifications.ts

```diff
@@ -0,0 +1,128 @@
+import {
+  buildNotifyTrigger,
+  type NotifyTrigger,
+  type NotifyTriggerState,
+} from '@coveo/headless';
+import {html, LitElement, nothing} from 'lit';
+import {customElement, property, state} from 'lit/decorators.js';
+import {map} from 'lit/directives/map.js';
+import {renderHeading} from '@/src/components/common/heading';
+import {bindStateToController} from '@/src/decorators/bind-state';
+import {bindingGuard} from '@/src/decorators/binding-guard';
+import {bindings} from '@/src/decorators/bindings';
+import {errorGuard} from '@/src/decorators/error-guard';
+import type {InitializableComponent} from '@/src/decorators/types';
+import {withTailwindStyles} from '@/src/decorators/with-tailwind-styles';
+import InfoIcon from '@/src/images/info.svg';
+import {InitializeBindingsMixin} from '@/src/mixins/bindings-mixin';
+import {AriaLiveRegionController} from '@/src/utils/accessibility-utils';
+import type {Bindings} from '../atomic-search-interface/interfaces';
+
+/**
+ * The `atomic-notifications` component is responsible for displaying notifications generated by the Coveo Search API (see [Trigger](https://docs.coveo.com/en/1458)).
+ *
+ * @part notifications - The wrapper around the notifications.
+ * @part notification - The wrapper around a single notification.
+ * @part icon - The icon of the notification.
+ * @part text - The text of the notification.
+ */
+@customElement('atomic-notifications')
+@bindings()
+@withTailwindStyles
+export class AtomicNotifications
+  extends InitializeBindingsMixin(LitElement)
+  implements InitializableComponent<Bindings>
+{
+  @state() public bindings!: Bindings;
+  @state() public error!: Error;
+
+  /**
+   * The [heading level](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements) to use above the notifications, from 1 to 6.
+   */
+  @property({type: Number, reflect: true, attribute: 'heading-level'})
+  headingLevel = 0;
+
+  /**
+   * Specifies an icon to display at the left-end of a notification.
+   *
+   * - Use a value that starts with `http://`, `https://`, `./`, or `../`, to fetch and display an icon from a given location.
+   * - Use a value that starts with `assets://`, to display an icon from the Atomic package.
+   * - Use a stringified SVG to display it directly
+   */
+  @property({type: String, reflect: true}) icon?: string;
+
+  @bindStateToController('notifyTrigger')
+  @state()
+  private notifyTriggerState!: NotifyTriggerState;
+  public notifyTrigger!: NotifyTrigger;
+
+  protected ariaMessage = new AriaLiveRegionController(this, 'notifications');
+
```

Comment:
I'd add prop validation (with throwOnError set to false, schema declared as private static readonly member of the class, and tests done similar to atomic-result-list.spec.ts, including TODO + Jira).